packages/atomic/src/utils/initialization-utils.tsx

```diff
@@ -129,15 +140,22 @@ export function InitializeBindings<SpecificBindings extends AnyBindings>() {
         (bindings: SpecificBindings) => {
           this.bindings = bindings;
 
-          const updateLanguage = () => forceUpdate(this);
+          const updateLanguage = () => forceUpdateComponent(this);
           this.bindings.i18n.on('languageChanged', updateLanguage);
           unsubscribeLanguage = () =>
             this.bindings.i18n.off('languageChanged', updateLanguage);
 
           try {
             // When no controller is initialized, updating a property with a State() decorator, there will be no re-render.
             // In this case, we have to manually trigger it.
-            this.initialize ? this.initialize() : forceUpdate(this);
+            if (this.initialize) {
```

Comment:
I think this might fix this issue here : https://discuss.coveo.com/t/ambiguous-error-atomic-icon-must-be-the-child-of-a-search-interface/8264

If so, would it be worth it to add the forceUpdate option to `atomic-icon`